src/winetricks

```diff
@@ -12968,7 +12968,7 @@ w_metadata vcrun2017 dlls \
     publisher="Microsoft" \
     year="2017" \
     media="download" \
-    conflicts="vcrun2015 vcrun2019" \
+    conflicts="vcrun2017 vcrun2019 ucrtbase2019" \
```

Comment:
Why did you remove vcrun2015, and why was vcrun2017 added? There's no need for it to conflict with itself.

Reply:
My mistake I'll fix that